src/main/java/uk/gov/hmcts/reform/sscs/config/CitizenCcdClient.java

```diff
@@ -87,5 +92,29 @@ StartEventResponse startEventForCitizen(IdamTokens idamTokens, String caseId, St
             eventType
         );
     }
+
+    void addUserToCase(IdamTokens idamTokens, String userIdToAdd, Long caseId) {
+        caseAccessApi.grantAccessToCase(
+                idamTokens.getIdamOauth2Token(),
+                idamTokens.getServiceAuthorization(),
+                idamTokens.getUserId(),
+                ccdRequestDetails.getJurisdictionId(),
+                ccdRequestDetails.getCaseTypeId(),
+                caseId.toString(),
+                new UserId(userIdToAdd)
+        );
+    }
+
+    void removeUserFromCase(IdamTokens idamTokens, String userIdToRemove, Long caseId) {
```

Comment:
Is this ever used? Can't see any references to it, or is it to future proof the code?

Reply:
yes its not use, but in case in future if we want to de-associate user